python/py-grpcio/Portfile

```diff
@@ -4,24 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-grpcio
-version             1.25.0
+version             1.35.0
 revision            0
 platforms           darwin
 license             Apache-2
 maintainers         {emcrisostomo @emcrisostomo} openmaintainer
 
 description         HTTP/2-based RPC framework
-long_description    ${description}
+long_description    {*}${description}
```

Comment:
I have never really understood why people are making this change in occasional (rare) ports. What does it fix?

Reply:
Thanks @kencu, that was my call. From long_description section of the [Global keywords](https://guide.macports.org/#reference.keywords) docs:

> Sometimes the port's name and short description are reused as part of the long description. When referencing the description keyword (or any other list keyword with more than one item), it should be preceded with the expand operator.

I'm no tcl expert, but my understanding is that it helps with expansion. This is also described in the [tcl wiki](https://wiki.tcl-lang.org/page/%7B%2A%7D).

Reply:
Ah. Well ${description} is just a string, right? Not a list. So there is no need for it, AFAICT. Which I guess is why it seems to make absolutely no difference when doing it :>

But I see it has been added to the guide, so there's that !

Reply:
That's a good point @kencu, I would've thought it's a string too, so I don't know.

I traced the addition of said text to https://github.com/macports/macports-guide/commit/ff6ebb3794be11092c5f8108dddcd5557381e386 which was added by @ryandesign. Maybe he can tell us where I'm going wrong.